tests/vmi_lifecycle_test.go

```diff
@@ -220,6 +220,9 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		})
 
 		DescribeTable("log libvirtd debug logs should be", func(vmiLabels, vmiAnnotations map[string]string, expectDebugLogs bool) {
+			arch := testsuite.TranslateBuildArch()
+
+			checks.SkipIfS390X(arch, "failing consistently for s390x.")
```

Comment:
Please make a review why certain tests fail.

Reply:
thanks for the review @xpivarc, sure i will make a review for the test failures.